lstchain/io/io.py

```diff
@@ -18,7 +18,9 @@
 from astropy.utils import deprecated
 from ctapipe.instrument import OpticsDescription, CameraGeometry, CameraDescription, CameraReadout, \
     TelescopeDescription, SubarrayDescription
-
+from pyirf.simulations import SimulatedEventsInfo
+from astropy import table
+import pandas as pd
 
 __all__ = ['read_simu_info_hdf5',
            'read_simu_info_merged_hdf5',
```

Comment:
Here 'read_dl2_to_pyirf' should be included as well

Reply:
And also in the init script

Reply:
Done

Reply:
(in init it's only necessary to expose the function in `lstchain.io` which is useful for external users - I don't think it's necessary here)

Reply:
Oh ok! Understood, thanks!